src/models/session.js

```diff
@@ -39,6 +39,18 @@ class Session {
   }
 
   /**
+   * Initializes session for user (if defined) in Store
+   * Note: This should be the FIRST call before attempting any other session operations
+   *
+   * @return {Promise}
+   *
+   */
+  initialize() {
+    return this._user.syncWithStore();
```

Comment:
Basically it's just a wrapper, but I believe that it makes the API cleaner. (it's cleaner to initialize the Session since the read-only clients will only perform operations on the Session)